File: bronzeonestar/Calculator/CalculatorUnitTest/UnitTest1.cs
using System;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using Calculator;

namespace CalculatorUnitTest
{
    [TestClass]
    public class UnitTest1
    {
        [TestMethod]
        public void TestMethod1()
        {
            CalculatorMakeup g = new CalculatorMakeup(5);
          Assert.AreEqual(CalculatorMakeup.testc(), 5);

        }
    }
}

File: Calculator/CalculatorUnitTest/unittest1.cpp
#include "stdafx.h"
#include "CppUnitTest.h"
#include"../Calculator/Calculator.h"


using namespace Microsoft::VisualStudio::CppUnitTestFramework;

namespace CalculatorUnitTest
{		
	TEST_CLASS(UnitTest1)
	{
	public:
		
		TEST_METHOD(TestMethod1)
		{
			// TODO: 在此输入测试代码
			Calculator c;
			Assert::AreEqual(true,c.Is_int(3));
		}

	};
}